source/_integrations/connectedcars.markdown

```diff
@@ -0,0 +1,36 @@
+---
+title: ConnectedCars.io
+description: Instructions on how to integrate ConnectedCars.io within Home Assistant.
+ha_category:
+  - Car
+  - Presence Detection
+  - Sensor
+ha_release: 0.112
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@heslegrave'
+ha_domain: connectedcars
+---
+
+The `connectedcars` integration uses the [ConnectedCars.io](https://connectedcars.io/) web service as a source for car data.
+This is mainly used by the Volkswagen group in Denmark and is the backside of [MinVolkswagen](https://site.volkswagen.dk/minvolkswagen/).
+
+Thanks to [niklascp](https://pypi.org/user/niklascp/) for creating the PyPi lib so I did not have to..
```

Comment:
Giving credit is nice, but we should do it at the end of the page.

Furthermore, don't write it from your perspective, instead, use a third-person view.